### PR TITLE
doc/installing.md: add installation steps for Gentoo

### PR DESCRIPTION
**Pull request by juippis**
Link to a Gentoo Wiki's main Incus page will be added once it's created. It'll hold more information, examples, Gentoo-specific use cases and so on.

### CODE REVIEW

File: doc/installing.md [hunk 1]
```diff
@@ -66,6 +66,49 @@ If you prefer a different installation method, see {ref}`installing`.
...
+
+ - **openrc**: edit /etc/conf.d/incus
+
+ - **systemd**: systemctl edit --full lxd.service
+
+Set up subuid and subgid:
+
+    echo "root:1000000:1000000000" | tee -a /etc/subuid /etc/subgid
+
+For more information: [`Gentoo Wiki / subuid/subgid`](https://wiki.gentoo.org/wiki/Subuid_subgid)
```
Comment (juippis) [inline doc/installing.md:91]: Wasn't sure if this or https://github.com/lxc/incus/blob/main/doc/userns-idmap.md was better to link here.

File: doc/installing.md [hunk 2]
```diff
@@ -66,6 +66,49 @@ If you prefer a different installation method, see {ref}`installing`.
...
+Note: Installing LTS vs. feature-release will be explained later, when Incus upstream and Gentoo's repository has those releases available.
+
+There will be two newly created groups associated to Incus: 
+`incus` to allow basic user access (launch containers), and `incus-admin` for `incus admin` controls. Add your regular users to either, or both, depending on your setup and use cases.
+
+After installation, you may want to configure Incus. This is optional though, as the defaults should also just work.
+
+ - **openrc**: edit /etc/conf.d/incus
+
+ - **systemd**: systemctl edit --full lxd.service
```
Comment (stgraber) [inline doc/installing.md:85]: @juippis that probably should be `incus.service`?

Comment (juippis) [reply doc/installing.md:85]: Oops, had to spawn a systemd vm which had lxd installed to check correct **systemctl** syntax. Sharp eyes!

File: doc/installing.md (hunk 1 above)
Comment (stgraber) [reply doc/installing.md:91]: I think so as most distributions will ahve the same "problem" when uidmap is in place on the system, so probably best to send everyone to the same package for consistency.

```
{ref}`{userns-idmap}`
```
should do the trick

Comment (juippis) [reply doc/installing.md]: Couldn't get that, or ``{ref}`userns-idmap`` ` to work. But this current way seems to.

**Comment by stgraber** [comment]
```
.tmp/doc/installing.md:86: MD005 Inconsistent indentation for list items at the same level
.tmp/doc/installing.md:87: MD005 Inconsistent indentation for list items at the same level
.tmp/doc/installing.md:97: MD005 Inconsistent indentation for list items at the same level
.tmp/doc/installing.md:98: MD005 Inconsistent indentation for list items at the same level
.tmp/doc/installing.md:86: MD006 Consider starting bulleted lists at the beginning of the line
.tmp/doc/installing.md:97: MD006 Consider starting bulleted lists at the beginning of the line
```

I'll fix that in place in your branch

**Comment by stgraber** [comment]
Thanks!

**Pull request merged**